docs/source/tts/data/ngc_models_aligner.csv

```diff
@@ -1,2 +1,3 @@
-Locale,Dataset,Sampling Rate,#Speakers,Model Name,Model Class,Model Card
-en-US,LJSpeech,22050Hz,Single,RAD-TTS Aligner,nemo.collections.tts.models.aligner.AlignerModel,https://ngc.nvidia.com/catalog/models/nvidia:nemo:tts_en_radtts_aligner
\ No newline at end of file
+Locale,Dataset,Sampling Rate,#Spk,Model Name,Phoneme Unit,Model Class,Overview,Checkpoint
```

Comment:
Nitpick: could you switch "Model Name" to be the second column and group them together like that? My guess is it's more likely that users will want to look for checkpoints by model rather than dataset.

Reply:
that's good point. will revise then.